### ISSUE TITLE
Space attributes

### ISSUE THREAD

**Issue by dantefromhell** [humhub/humhub#5789]
For user accounts `Profile attributes` can be configured by the Administrator + they are available to use as filters in the `People` menu.

For a larger instances with many Spaces this functionality would be very helpful.
E.g. for global Community platforms that use Spaces to organized localized IRL groups. The country attribute would allow to easily filter all Spaces/ Local groups within a Country.

If implemented through the same configurable pattern that is used for `Profile attributes` it would enable a flexible categorization and inventory of Spaces.

**Comment by lisandi** [comment]
That is a great idea and partly already existing actually when adding what is available in user location map https://www.humhub.com/de/marketplace/usermap/ and https://www.humhub.com/de/marketplace/members-map/ but specified for all spaces. In other words a space would have also a location and that location could (must be) country, province, City, long/lat. With that information, visitors could find spaces that are near their user location specified in a user/space location map. We checked both available apps but unfortunately both only focus on the user actually and NOT on a Space Location! So having a Space Location available would make things definitely much easier.

**Comment by dantefromhell** [comment]
I agree on @lisandi Location description. But I wanna make sure to state that my initial idea is not about allow Location only - it was merely the example I used.
Another (recent example) was the question of how one can associate Project Space with UN Sustainable Development Goals and than filter for projects centred around one or more of these goals.

The best implementation would be as universably configurable as attributes in the user profile.

**Comment by Gilbertdelyon** [comment]
@dantefromhell 
4 years later I had the [same idea](https://community.humhub.com/content/perma?id=295653)  and just discovered this thread.
My main goal would be to establish a link between certain events and certain spaces.
